pkg/config/validation/validation.go

```diff
@@ -1983,6 +1983,12 @@ func validateHTTPRoute(http *networking.HTTPRoute) (errs error) {
 		}
 	}
 
+	if http.MirrorPercent != nil {
+		if value := http.MirrorPercent.GetValue(); value > 100 {
```

Comment:
What if percent of value 0. Is it allowed?

Reply:
Yes. We disable the mirroring if value is 0.